### DISCUSSION TITLE
Remove boilerplate at the top of capabilities?

### DISCUSSION THREAD

**Discussion by davidstanke** [dora-team/dora.dev#167] [General]
Hi @michelleirvine! The capabilities pages were previously hosted on cloud.google.com, which is a much broader context than just DORA. So they each have an intro paragraph like the one below. Now that they're hosted in a DORA-specific context, I feel that's no longer necessary, so I propose removing it. What do you think?

_I propose that we remove these bits from the capability pages:_
`Note: Architecture is one of a set of capabilities that drive higher software delivery and organizational performance. These capabilities were discovered by the [DORA State of DevOps research program](http://localhost:1313/), an independent, academically rigorous investigation into the practices and capabilities that drive high performance. To learn more, read our [DevOps resources](https://cloud.google.com/devops).`

**Comment by michelleirvine** [comment]
Agree - you can take those out!

**Reply by davidstanke** [reply]
Sweet -- will do.

**Reply by mukteshkrmishra** [reply]
Is there any issue corresponding to this? @davidstanke

**Comment by davidstanke** [comment]
@mukteshkrmishra there is! Odd that it doesn’t cross-link to here, but it’s #176 — I’ll assign it to you. (If you don’t want it, just LMK)

**Comment by davidstanke** [comment]
Oh wait, I’ll need you to comment on it before I can assign you to it. So if you do want it, just say something on #176. Thanks!
